cmd/skupper/skupper.go

```diff
@@ -440,10 +441,26 @@ installation that can then be connected to other skupper installations`,
 				return fmt.Errorf("The --enable-flow-collector option must be used with the --enable-console option")
 			}
 
+			if routerCreateOpts.AuthMode != "internal" && (len(routerCreateOpts.User) > 0 || len(routerCreateOpts.Password) > 0) {
+				return fmt.Errorf("the --console-auth option must be set to internal")
```

Comment:
Maybe clarify that the reason for this is the presence of username/password?

Reply:
That makes sense, thanks!

Reply:
Updated in b2bd80797a311e98cad54e4793baed3b2244ef91